.gitignore

```diff
@@ -1,3 +1,5 @@
 /backup_all_my_flickr_photos.egg-info
 /dist
 /build
+/Download*
+.python-version
```

Comment:
If the user is using `pyenv`, they can ignore `.python-version` by listing it in `.git/info/exclude`, which isn't committed to the repo. There is no need to add it to the repo that is shared by everyone.